admin/pages/tools.php

```diff
@@ -26,7 +26,7 @@
 			'desc' => __( 'Import settings from other SEO plugins and export your settings for re-use on (another) blog.', 'wordpress-seo' ),
 		),
 	);
-	if ( WPSEO_Utils::allow_system_file_edit() === true && ! is_multisite() ) {
+	if ( WPSEO_Utils::allow_system_file_edit() === true && ( ! is_multisite() || is_network_admin() ) ) {
```

Comment:
I don't understand what you are trying to do here. This condition is meant for siteadmins. On a single site install they should be able to access the file editor through the tools settings page.